espnet/asr/pytorch_backend/asr.py

```diff
@@ -235,8 +224,8 @@ def __init__(self, subsampling_factor=1, dtype=torch.float32):
         self.ignore_id = -1
         self.dtype = dtype
 
-    def __call__(self, batch, device):
-        """Transforms a batch and send it to a device.
+    def __call__(self, batch, device=None):
```

Comment:
Maybe this one is more clear?
```suggestion
    def __call__(self, batch, device="cpu"):
```

Reply:
If so, you can revert `to_func`.